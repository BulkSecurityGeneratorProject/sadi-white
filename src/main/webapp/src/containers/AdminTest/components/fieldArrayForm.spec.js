
// Copyright 2004-present Facebook. All Rights Reserved.

/* eslint-disable no-unused-vars */

import React from 'react';
import FieldArrayForm from './fieldArrayForm'
import { reducer as formReducer } from 'redux-form'
import { createStore, combineReducers } from 'redux'
import {Provider} from 'react-redux'
import {Button} from 'react-bootstrap';

import renderer from 'react-test-renderer';

const reducer = combineReducers({
  form: formReducer,
})

const store = createStore(reducer)

describe('fieldArrayForm', () => {
  it('should render', () => {
    const form = renderer
      .create(<Provider store={store}><FieldArrayForm/></Provider>)
      .toJSON();

    expect(form).toMatchSnapshot();
  });
})
