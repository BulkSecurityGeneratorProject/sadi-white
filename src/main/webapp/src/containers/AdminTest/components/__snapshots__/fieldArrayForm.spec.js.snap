// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fieldArrayForm should render 1`] = `
<form
  className=""
  onSubmit={[Function]}
>
  <div
    className="form-group"
  >
    <label
      className="control-label"
      htmlFor={undefined}
    >
      Test Name
    </label>
    <input
      className="form-control"
      id={undefined}
      name="name"
      onBlur={[Function]}
      onChange={[Function]}
      onDragStart={[Function]}
      onDrop={[Function]}
      onFocus={[Function]}
      type="text"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <label
      className="control-label"
      htmlFor={undefined}
    >
      Test Duration
    </label>
    <input
      className="form-control"
      id={undefined}
      name="duration"
      onBlur={[Function]}
      onChange={[Function]}
      onDragStart={[Function]}
      onDrop={[Function]}
      onFocus={[Function]}
      type="text"
      value=""
    />
  </div>
  <div
    className="form-group"
  >
    <div
      className="col-md-2"
    >
      <button
        className="btn btn-primary"
        disabled={false}
        onClick={[Function]}
        type="button"
      >
        Add Questions
      </button>
    </div>
    <div
      className="col-md-10"
    />
  </div>
  <div
    className="clearfix"
  />
</form>
`;
